pkg/mount/mount.go

```diff
@@ -47,8 +49,8 @@ const mountpointPerms os.FileMode = 0700
 // It checks the volume first, waiting 30 seconds for it to be created, and
 // creates an ext4 filesystem on it if there isn't already a filesystem.  The
 // mount will fail if the mount command can't determine the fstype.
-func mountVolume(ctx context.Context, deviceRootPath string, id string, mp string, fsType string) error {
-	if err := initRawVolume(ctx, deviceRootPath+"/"+id, fsType); err != nil {
+func mountVolume(stderr io.Writer, ctx context.Context, deviceRootPath string, id string, mp string, fsType string) error {
+	if err := initRawVolume(stderr, ctx, deviceRootPath+"/"+id, fsType); err != nil {
```

Comment:
by convention ctx should be first param, perhaps best as last param, and called "errwriter" or similar (it might not get passed stderr)?